tests/test_samples.py

```diff
@@ -911,3 +920,19 @@ def test_logL_list():
 
     samples = NestedSamples(data=data, logL=logL, logL_birth=logL_birth)
     assert_array_equal(default, samples)
+    plt.close("all")
+
+
+def test_hist_plotting():
+    samples = NestedSamples(root='./tests/example_data/pc')
+    samples[['x0', 'x1', 'x2', 'x3', 'x4']].plot.hist()
+    samples.x0.plot.kde(subplots=True)
+
+    samples.plot.hist_2d('x0', 'x1')
+    samples.plot.kde_2d('x0', 'x1')
+    samples.plot.fastkde_2d('x0', 'x1')
+    samples.plot.kde_1d()
+    samples.plot.fastkde_1d()
+    samples.plot.hist_1d()
+
+    plt.close("all")
```

Comment:
* Does this belong here? Or rather in `test_plot.py` or even `test_weighted_pandas.py`?
* Should these tests use a `WeightedDataFrame` instead of `NestedSamples` as a baseline?
* Can we make these tests a bit more meaningful, beyond just testing the existence of those methods?

Reply:
> Does this belong here? Or rather in `test_plot.py` or even `test_weighted_pandas.py`?
> Should these tests use a `WeightedDataFrame` instead of `NestedSamples` as a baseline?

Renamed to `test_samples_dot_plot` to make clear these tests are meant primarily to test for the pandas `.plot` attribute. These are not meant to be thorough tests of the plotting functions.

> Can we make these tests a bit more meaningful, beyond just testing the existence of those methods?

Done in [aa24d72](https://github.com/williamjameshandley/anesthetic/pull/207/commits/aa24d724a9e7f6ab005b3e441b5394325b2165d0).